.changeset/young-ducks-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Changed] - System tests to Playwright system tests
```

Comment:
probably need a bit more detail here and say we're replacing existing tests written using wdio to use playwright